src/cryptoadvance/specter/templates/includes/helpers.jinja

```diff
@@ -1,26 +1,189 @@
 <!-- This file contains javascript helper functions that also can use jinja -->
 <script type="text/javascript">
 
-    // Formats an unitLabel (e.g. from a tx output) to a useful unit label
-    function formatUnitLabel(unitLabel, convertToSat){
-        var newLabel = unitLabel;
+
+
+    /*
+        Formats the btc amount as sats. 
+        Args:
+            value (Union[float, str]): (in BTC) Will convert string to float. 
+            enableDigitFormatting (bool, optional): Will group the Satoshis into blocks of 3,
+                e.g. 3 123 456, and color the blocks. Defaults to false.
+        Returns:
+            str: The formatted btc amount as html code.
+        */
+    function internalFormatBtcAmountAsSats(
+        value,
+        enableDigitFormatting=false,
+    ) {
+        const locale = 'en-US';
+        if (isNaN(parseFloat(value))){
```

Comment:
Not sure I understand the purpose of this if clause, what do you want to achieve here?

Reply:
If it is NaN, it should stop trying to process the number.

Reply:
OK, I see, but wouldn't it be better to return an actual error then (instead of NaN basically)?

Reply:
Yes, you are right.  Done in  ab44af63  . And I now notice that I actually don't use this case...